README.md

```diff
@@ -4,27 +4,19 @@ Confluent's Python Client for Apache Kafka<sup>TM</sup>
 **confluent-kafka-python** is Confluent's Python client for [Apache Kafka](http://kafka.apache.org/) and the
 [Confluent Platform](https://www.confluent.io/product/compare/).
 
-Features:
+We provide high-level Producer, Consumer and AdminClient classes analogous to the Java client. These are backwards and forwards compatible with all Kafka brokers from version 0.8.
```

Comment:
we is vague, 'confluent-kafka-python provides a high-level Pro..` ? Or Confluent .. ?

Reply:
I iterated on the wording - see what you think. found this remarkably difficult to write...